src/access/moderator-access.js

```diff
@@ -0,0 +1,64 @@
+// 'use strict'
+const type = 'moderator-access'
+
+const moderator = 'moderator'
+const member = 'member'
+
+class ModeratorAccessController {
+  constructor (rootMod, options) {
+    this._capabilityType = {}
+    this._capabilityType[moderator] = moderator
+    this._write = []     // Allowed to add other mods or members
+    this._rootMod = rootMod
+    this._write.push(this._rootMod)
+    this._members = !!options.members
```

Comment:
Boolean(options.members) is clearer!

Reply:
but bang bang is more fun!  changed it though